### PR TITLE
WIP updated spanish translation

### PR DESCRIPTION
**Pull request by asoltys**

### CODE REVIEW

File: src/locales/es.json [hunk 1]
```diff
@@ -1,275 +1,282 @@
...
-		"empty": "No hay transacciones.",
-		"send": "Enviar",
-		"next": "Próxima",
-		"sendersChoice": "Elección del remitente",
+		"sent": "Enviado",
+		"received": "Recibido",
+		"empty": "Aún no hay transacciones.",
+		"send": "Enviado",
+		"next": "Siguiente",
+		"sendersChoice": "Sender's choice",
```
Comment (mrxcell) [inline src/locales/es.json:256]: Line 256, change **"Sender's choice"** for **"Elección del remitente"**

**Review by mrxcell — APPROVED** [review]
Hi Fellows, somehow i missed Line 256 for translation, since i don´t code i´m afraid i might brake something LOL.

Please approve this last change!

Regards!

**Comment by ahopk127** [comment]
> Hi Fellows, somehow i missed Line 256 for translation, since i don´t code i´m afraid i might brake something LOL.
> 
> Please approve this last change!
> 
> Regards!

I'm the person who coded this translation system & wrote the manuals.  If you miss anything, the system will fallback to the English translation.  I'll add that to the manual.

**Comment by asoltys** [comment]
thank you!

**Pull request merged**